src/BibleWell.Aquifer.Data/Constants.cs

```diff
@@ -0,0 +1,17 @@
+using BibleWell.Storage;
+
+namespace BibleWell.Aquifer.Data;
+
+internal sealed class Constants(IStorageService _storageService)
```

Comment:
After looking through the usages, I suggest making this class `static` and then changing the `ApplicationPath` and `DatabasePath` properties into methods instead which accept a `string appDataDirectory`.  All of the callers have a storage service, anyway.

e.g.
```
private string GetApplicationPath(string appDataDirectory) => Path.Combine(appDataDirectory, ApplicationDir);
```